libraries/AP_HAL_ChibiOS/Util.cpp

```diff
@@ -375,19 +375,20 @@ Util::FlashBootloader Util::flash_bootloader()
 bool Util::get_system_id(char buf[40])
```

Comment:
Need to also increase this buffer by 10 so we don't cut of the ID.
```suggestion
bool Util::get_system_id(char buf[50])
```

Reply:
fixed, rebase/squashed.